styles/team.css

```diff
@@ -10,12 +10,12 @@
     display: flex;
     padding: 10rem 0;
 }
-.team-members__card {
+.team-members__list {
     flex: 1;
     text-align: center;
 }
-.team-members__job {
-    margin-top: 4rem;
+.team-members__item.heading__small {
```

Comment:
W zasadzie sama klasa `.heading__small` powinna wystarczyć.
Jeśli chcesz coś wyróżnić to użyj modyfikatora np. `heading__small heading__small--more-space`